api/delete-text.md

```diff
@@ -41,8 +64,52 @@ In this example:
  * `example` : Metric name
  * `1527811200` : Delete all data for this metric before this time
 
-### Output
+### Sample Output for Single Metric Example
 
 ```
 []
 ```
+
+## Wildcard Metric Example
+
+```
+curl -X DELETE \
+     -H 'x-snowth-delete-time: 1527811200' \
+     -H 'x-snowth-account-id: 1234' \
+     -H 'x-snowth-confirm-delete: 1' \
+     http://127.0.0.1:8112/text/6f6bdc73-2352-4bdc-ab0e-72f66d0dee12/multiple_example*
+```
+
+In this example:
+
+ * `text` : This tells the system that text data will be removed.
+ * `6f6bdc73-2352-4bdc-ab0e-72f66d0dee12` : Check UUID
+ * `multiple_example*` : Metric name pattern including wildcard
+ * `1527811200` : Delete all data for matching metrics before this time
+ * `1234` : Delete data only for the given account id
+ * `1` : Confirm to actually commit to the deletion (we highly recommend omitting this header at first, to examine what will be deleted)
+
+### Sample Output for Wildcard Metric Example
+
+```
+[ {"metric_name":"multiple_example_cpuutil_server1","delete_result":"not local","payload":""},
+  {"metric_name":"multiple_example_cpuutil_server2","delete_result":"deleted","payload":""},
+  ...
+]
+```
+
+### Wildcard, Tag Query and Check Delete Statuses
```

Comment:
Thanks for adding this section, it will be really helpful. I noticed though, that in the example above, the second result is "deleted" but that doesn't appear in your list of result/status values. Oversight?

Reply:
good catch, for deletion the code changes "ok" to "deleted" or "found" depending on whether delete confirmation was given in the header.  I have fixed this.
I also moved the list into a separate page because I had forgotten to add it to check deletes too, and otherwise it would be the same text block copied in 8 places.